[2024-10-03T17:13:48.221+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T17:13:48.228+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T11:43:35.551176+00:00 [queued]>
[2024-10-03T17:13:48.233+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T11:43:35.551176+00:00 [queued]>
[2024-10-03T17:13:48.233+0530] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-03T17:13:48.254+0530] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): validate_data> on 2024-10-03 11:43:35.551176+00:00
[2024-10-03T17:13:48.259+0530] {standard_task_runner.py:72} INFO - Started process 25831 to run task
[2024-10-03T17:13:48.262+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_ingestion_dag', 'validate_data', 'manual__2024-10-03T11:43:35.551176+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmpzrn7eomo']
[2024-10-03T17:13:48.264+0530] {standard_task_runner.py:105} INFO - Job 62: Subtask validate_data
[2024-10-03T17:13:48.309+0530] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T11:43:35.551176+00:00 [running]> on host Nithesh-Kumar.localdomain
[2024-10-03T17:13:48.385+0530] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_ingestion_dag' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-03T11:43:35.551176+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-03T11:43:35.551176+00:00'
[2024-10-03T17:13:48.386+0530] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T17:13:48.451+0530] {file_data_context.py:222} INFO - FileDataContext loading fluent config
[2024-10-03T17:13:48.456+0530] {config.py:187} INFO - Loading 'datasources' ->
[]
[2024-10-03T17:13:48.578+0530] {validator.py:1467} INFO - 	28 expectation(s) included in expectation_suite.
[2024-10-03T17:13:48.816+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T17:13:48.817+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T17:13:48.819+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:00, 1275.06it/s]
[2024-10-03T17:13:48.872+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:01, 36.25it/s]  
[2024-10-03T17:13:48.873+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:01, 53.77it/s]
[2024-10-03T17:13:48.909+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:02, 32.72it/s]
[2024-10-03T17:13:48.920+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 164.96it/s]
[2024-10-03T17:13:48.920+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 164.96it/s]
[2024-10-03T17:13:48.951+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 164.96it/s]
[2024-10-03T17:13:48.983+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 164.96it/s]
[2024-10-03T17:13:48.996+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 164.96it/s]
[2024-10-03T17:13:48.996+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 164.96it/s]
[2024-10-03T17:13:48.996+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 406.41it/s]
[2024-10-03T17:13:49.338+0530] {logging_mixin.py:190} INFO - Data quality check passed.
[2024-10-03T17:13:49.884+0530] {python.py:240} INFO - Done. Returned value was: None
[2024-10-03T17:13:49.889+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T17:13:49.889+0530] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_ingestion_dag, task_id=validate_data, run_id=manual__2024-10-03T11:43:35.551176+00:00, execution_date=20241003T114335, start_date=20241003T114348, end_date=20241003T114349
[2024-10-03T17:13:49.922+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-03T17:13:49.935+0530] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-03T17:13:49.949+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
